homework3/tests/test_wrapper.py

```diff
@@ -0,0 +1,14 @@
+from typing import List, Tuple
+
+import pytest
+
+from homework3.task1.cache_wrapper import f
+
+
+def test_decor_cache(monkeypatch):
+    monkeypatch.setattr("builtins.input", lambda x: 1)
```

Comment:
Actually, this test funtion does not test caching. Could you improve it, please?